classification.py

```diff
@@ -1262,8 +1263,8 @@ def highlight_max(s):
         sample_size = input("Sample Size: ")
         
         if sample_size == '' or sample_size == '1':
-            
-            X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=1-train_size, stratify=y, random_state=seed, shuffle=data_split_shuffle)
+            stratify = y if data_split_shuffle else None

```

Comment:
Yes, I you have a look at the changes in the latest commit, I have implemented option 1.
If the `shuffle` parameter is set to `True`, the `stratify` is automatically set to `None` to work with the default `sklearn` behavior. This is also updated in the documentation.